npc/common/src/main/java/org/screamingsandals/lib/npc/NPCImpl.java

```
@@ -60,8 +65,27 @@
     private volatile boolean lookAtPlayer;
     private final List<MetadataItem> metadata;
 
+    
     private SClientboundSetPlayerTeamPacket.CollisionRule collisionRule;
-
+    private static boolean IS_BUNGEECORD_ENABLED;
+    static {
+        File spigotYmlFile = new File("spigot.yml");
+        IS_BUNGEECORD_ENABLED = false;
+        java.util.regex.Pattern pattern = Pattern.compile("bungeecord: ?(y|yes|true|on)",Pattern.CASE_INSENSITIVE);
+
+        if (PluginManager.getPlatformType() == PlatformType.BUKKIT && spigotYmlFile.isFile()) {
+            try (Scanner scanner = new Scanner(spigotYmlFile)) {
+                while (scanner.hasNextLine()) {
+                    String line = scanner.nextLine().toLowerCase();
+
+                    if (pattern.matcher(line).matches())
+                        IS_BUNGEECORD_ENABLED = true;
+                }
+            } catch (FileNotFoundException e) {
```

fail silently instead of printing a stacktrace